src/Controls/src/Core/TabbedPage/TabbedPage.Windows.cs

```diff
@@ -325,6 +357,7 @@ internal static void MapItemsSource(ITabbedViewHandler handler, TabbedPage view)
 					(vm, page) =>
 					{
 						vm.Icon = page.IconImageSource?.ToIconSource(handler.MauiContext!)?.CreateIconElement();
+						vm.IconColor = (page.IconImageSource as FontImageSource)?.Color?.AsPaint()?.ToPlatform();
```

Comment:
Could include an UITest and validate the colors with the `VerifyScreenshot` method?

Reply:
Yes, in this PR, remove the **#if TEST_FAILS_ON_WINDOWS** compilation directive from the FontImageSourceColorShouldApplyOnTabIcon test and commit the snapshot to validate the test.